pkg/velero/syncManager.go

```diff
@@ -0,0 +1,131 @@
+package velero
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/cloudogu/k8s-backup-operator/pkg/retry"
+
+	backupv1 "github.com/cloudogu/k8s-backup-operator/pkg/api/v1"
+	velerov1 "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type defaultSyncManager struct {
+	veleroClientSet    veleroClientSet
+	ecosystemClientSet ecosystemClientSet
+	recorder           eventRecorder
+	namespace          string
+}
+
+// SyncBackups syncs backup CRs with velero CRs
+func (d *defaultSyncManager) SyncBackups(ctx context.Context) error {
```

Comment:
🚫 **[checkstyle]** <funlen> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Function 'SyncBackups' is too long (66 > 60)